common/models/update_service_request.go

```diff
@@ -1,5 +1,44 @@
 package models
 
+import (
+	"fmt"
+
+	swagger "github.com/zpatrick/go-plugin-swagger"
+)
+
 type UpdateServiceRequest struct {
-	DeployID string `json:"deploy_id"`
+	ServiceID         string  `json:"service_id"`
+	DeployID          *string `json:"deploy_id"`
+	ServiceScaleCount *int    `json:"service_scale_count"`
```

Comment:
[nit] `ServiceScaleCount` seems like a lot - can we just do `Scale`?

Reply:
✅